Command/SwarrotCommand.php

```diff
@@ -106,7 +106,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
      */
     protected function getOptions(InputInterface $input)
     {
-        $options = $this->extras + [
```

Comment:
Having a second look, in order to allow CLI arguments to override the processors default, shouldn't we do something like that ?
```php
        $options = [];

        // Processors options
        foreach ($this->processorConfigurators as $processorConfigurator) {
            $processorOptions = $processorConfigurator->resolveOptions($input);
            if ($processorConfigurator->isEnabled()) {
                $options += $processorOptions;
            }
        }

        // Consumer configuration options (from symfony config)
        $options += $this->extras;

        // CLI arguments
        $options += [
            'queue' => $input->getArgument('queue'),
            'connection' => $input->getArgument('connection'),
            'poll_interval' => (int) $input->getOption('poll-interval'),
        ];
```

Reply:
i would leave the cli arguments at the beginning, at least the two required arguments (queue and connection), so they can't be overriden by mistake

Reply:
Oops forget my previous comment ! I never use the `+=` array syntax and get confused ^^

In my opinion
- CLI should override Symfony Config & Processor defaults
- Symfony config should override Processor defaults

```php
        // CLI arguments
        $options = [
            'queue' => $input->getArgument('queue'),
            'connection' => $input->getArgument('connection'),
            'poll_interval' => (int) $input->getOption('poll-interval'),
        ];

        // Consumer configuration options (from symfony config)
        $options += $this->extras;

        // Processors defaults
        foreach ($this->processorConfigurators as $processorConfigurator) {
            $processorOptions = $processorConfigurator->resolveOptions($input);
            if ($processorConfigurator->isEnabled()) {
                $options += $processorOptions;
            }
        }
```

Does it make sense ?

Reply:
sry for the delay,
actually you can't if consumer configuration options are set before processors defaults, we can't override the config with command options (this is almost the code before this pr).
The last loop is not exactly processors defaults, its the the processor value from the command line, and if they are not overriden in the command line it will take the default argument of the command, often beeing $this->getExtras('my_extra', default) (ie see https://github.com/swarrot/SwarrotBundle/blob/master/Processor/MaxExecutionTime/MaxExecutionTimeProcessorConfigurator.php#L53)

Reply:
ping @olaurendeau :p

Reply:
Oops sorry for the delay @raphael-trzpit 
I'd like to have @odolbeau opinion on this one, has it seems around today :)